### ISSUE TITLE
Add to Bucket browser plugin: Checkbox user interface is misleading.

### ISSUE THREAD

**Issue by dan-king** [ushahidi/SwiftRiver#301]
The 'Add to Bucket' browser plugin is a handy feature and super easy to use. Unfortunately ,the checkbox user interface belies how easy it is to use. 

Simply checking a box on or off adds or removes the page to the given bucket. But the checkbox interface implies there is a submit button that needs to be clicked. (Indeed, I keep incorrectly clicking the submit button next to the 'new bucket' input field).

It is not intuitive that you must click 'Close' or 'X' immediately after clicking a checkbox on or off.

A more intuitive interface would be to replace the checkboxes with buttons that say "Add to bucket" and "Remove from bucket" with an animated effect on each click. Also consider changing the text of the "Close" link to a "Done" button.

**Comment by 69mb** [comment]
How about the pseudo-checkboxes used for following/unfollowing on the dashboard shown below? Clicking them shows a loading icon. Otherwise @brandonrosage your thoughts?

![](https://gist.github.com/raw/f28ceade9732895ad1d9/e4f152ec67cc033ef3c752f69cfabb4545934595/WkCXXUa5HPx137i6.png)

**Comment by brandonrosage** [comment]
Yes, @69mb and @dan-king. I think it's absolutely right to unify the UI for selecting/deselecting items with both a label (e.g. checkmark icon or "Add") and unique visual state for selected items.

Fortunately, I've already started to incorporate this pattern in my next wave of UI improvements. Take for example the modal window for "Add collaborator."

![Add collaborator](http://f.cl.ly/items/2Y1c383p1G0k3Q0Q2E0O/Screen%20Shot%202012-11-29%20at%205.23.58%20PM.png)

And for consistency, much of this 'selected visual state' pattern is borrowed for table views, like your Dashboard "Summary." Though it uses checkboxes to perform more complex group actions, many of the same visual cues are there.

![Dashboard Summary](http://f.cl.ly/items/2N3y0b2I081F1W052B3n/Screen%20Shot%202012-11-29%20at%205.24.23%20PM.png)
